src/app/data-model-provider/Provider.h

```diff
@@ -66,18 +68,12 @@ class Provider : public ProviderMetadataTree
     ///         > Else if reading from the attribute in the path requires a privilege that is not
     ///           granted to access the cluster in the path, then the path SHALL be discarded.
     ///
-    /// Return codes:
-    ///   CHIP_ERROR_NO_MEMORY or CHIP_ERROR_BUFFER_TOO_SMALL:
+    /// Return value notes:
+    ///   ActionReturnStatus::IsOutOfSpaceError(i.e. CHIP_ERROR_NO_MEMORY or CHIP_ERROR_BUFFER_TOO_SMALL):
```

Comment:
Again, we should not be claiming specific error codes here, ideally.  Or at least remove that in a followup when we fix the representation.

Reply:
Removed details.